testsuite/integration-arquillian/tests/base/src/test/resources/run-on-server-jboss-deployment-structure.xml

```diff
@@ -12,6 +12,7 @@
             <module name="org.keycloak.keycloak-model-jpa"/>
             <module name="org.keycloak.keycloak-kerberos-federation"/>
             <module name="org.keycloak.keycloak-ldap-federation"/>
+	    <module name="org.infinispan"/>
```

Comment:
Padding is 2 spaces instead of 4. Very minor thing, just mentioning it for the future :)